bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java

```diff
@@ -118,27 +118,30 @@ public void initialize() {
 
         impactedModules.clear();
 
-        try {
-            ThingUID bridgeUID = thing.getBridgeUID();
-            if (bridgeUID == null) {
-                throw new IllegalArgumentException("Bridge does not exist!");
+        Object impactedModulesObject = getConfig().get(CONFIG_IMPACTED_MODULES);
+        if (impactedModulesObject != null) {
+            try {
+                ThingUID bridgeUID = thing.getBridgeUID();
+                if (bridgeUID == null) {
+                    throw new IllegalArgumentException("Bridge does not exist!");
+                }
+
+                String[] impactedModulesString = impactedModulesObject.toString().split(",");
+                for (String impactedModuleString : impactedModulesString) {
+                    ImpactedModuleUID impactedModuleUID = new ImpactedModuleUID(impactedModuleString.trim());
+                    ThingTypeUID thingTypeUID = new ThingTypeUID(bridgeUID.getBindingId(),
+                            impactedModuleUID.getThingTypeId());
+                    ThingUID thingUID = new ThingUID(thingTypeUID, bridgeUID, impactedModuleUID.getThingId());
+                    impactedModules.add(new ImpactedModule(thingUID, impactedModuleUID.getGroup()));
+                }
```

Comment:
Is it expected for the impactedModules to contain ThingUIDs for things that might not actually exist?
What is the expected behavior if the config contains a thing that doesn't exist?
You can check the validity of a thingUID by iterating through all of the things under a bridge using `getBridge().getThings()`.

That said I don't know what your requirements are here.

Reply:
Great suggestion, fixed!